mappings/net/minecraft/block/entity/BeaconBlockEntity.mapping

```diff
@@ -36,6 +36,8 @@ CLASS net/minecraft/unmapped/C_wwadquuj net/minecraft/block/entity/BeaconBlockEn
 		ARG 1 x
 		ARG 2 y
 		ARG 3 z
+	METHOD m_jlfysuud getEffectOrNull (Lnet/minecraft/unmapped/C_jaqasomh;)Lnet/minecraft/unmapped/C_jaqasomh;
```

Comment:
```suggestion
	METHOD m_jlfysuud getEffect (Lnet/minecraft/unmapped/C_jaqasomh;)Lnet/minecraft/unmapped/C_jaqasomh;
```
I don't think `OrNull` is needed here, `get(...)` methods typically return null if the thing can't be gotten

Reply:
hm, am not sure about doing that

Reply:
I support the change, `getEffectNullable` is better than `orNull` if you're still not a fan

Reply:
i think `OrNull` is fine